[SVN] Matlab-plugin: installation, configuration and demo

Matthias-A-Steiner opened the pull request.

Matthias-A-Steiner committed:
Create README.md

Matthias-A-Steiner committed:
Add files via upload

Matthias-A-Steiner committed:
Update README.md

Matthias-A-Steiner commented:
Here the changed and the new files in zip

Matthias-A-Steiner commented:
Here the changed and the new files in zip

Matthias-A-Steiner committed:
Add files via upload

da-liii commented:
Please use Git and put these files under the proper directories.

Thanks for your contribution!

Commit:
geändert:       README.md
	geändert:       bin/tm_matlab
	umbenannt:      matlab-abstract.en.tm -> doc/matlab-abstract.en.tm
	umbenannt:      matlab-demo.en.tm -> doc/matlab-demo.en.tm
	umbenannt:      matlab-install.en.tm -> doc/matlab-install.en.tm
	geändert:       doc/matlab.en.tm
	gelöscht:       matlab.en.tm
	gelöscht:       tm_matlab

Matthias-A-Steiner commented:
Dear Darcy,

I used Git, and I hope the files are now there where they should be and visible for review. Please, let me know, if I made any mistakes.

Tilda

> Am 05.03.2021 um 12:12 schrieb Darcy Shen <notifications@github.com>:
> 
> 
> Please use Git and put these files under the proper directories.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/texmacs/matlab/pull/1#issuecomment-791352415>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AO5K64WXIW5DH3CB7BL4RPDTCC4AVANCNFSM4YT2DAGQ>.
>

da-tubi commented on doc/matlab.en.tm:
```diff
@@ -1,11 +1,21 @@
+    <branch|Example session|matlab-demo.en.tm>
+  </traverse>
+
+  <tmdoc-copyright|2021|Tilda <abbr|A.> Steiner>
```
It would be better to reserve Joris's name. And the copyright year should be `2011-2021` which indicates the history of the doc.

da-liii replied:
It would be better to reserve Joris's name. And the copyright year should be 2011-2021 which indicates the history of the doc.

sadhen commented on doc/matlab-install.en.tm:
```diff
@@ -0,0 +1,128 @@
+  <verbatim|startup.m>, if there is one, will be considered. <em|Now> you're
+  up and running.
+
+  <tmdoc-copyright|2021|Tilda <abbr|A.> Steiner>
```
It would be better to reserve Joris's name. And the copyright year should be 2011-2021 which indicates the history of the doc.

da-liii commented:
`.DS_Store` is generated under macOS. Remove it and use `.gitignore` to avoid adding it again.

Googling about gitignore, and you will find what gitignore is used for.

Commit:
neue Datei:     .gitignore
 	geändert:       doc/matlab-install.en.tm
 	geändert:       doc/matlab.en.tm

da-liii commented:
I need to install Matlab first to verify if this PR works.

I'm not a matlab user. And I only have a 30-day free trial.

I will test your PR and try to port the improvements from Octave to Matlab later when I have more free time.

da-liii commented:
I tried to install matlab but failed to download matlab in China. I review the PR and found that only `bin/tm_matlab` in the code part is changed. I trust you, so I will merge this PR without actually launching the matlab plugin by myself.

> I will test your PR and try to port the improvements from Octave to Matlab later when I have more free time.

If you are interested in porting the improvements from the Octave plugin to the Matlab plugin. Just create PRs to this repo. I'm not a matlab user and failed to install a trial Matlab for reviewing PRs. I wonder if @pireddag would help verify (actually using the matlab plugin) the correctness of other PRs(if any). Or you can provide screenshots to demonstrate your changes.

da-liii commented:
One more tips for your PR, using

```
git rm .DS_store
```
to delete it.

pireddag commented:
> I tried to install matlab but failed to download matlab in China. I review the PR and found that only `bin/tm_matlab` in the code part is changed. I trust you, so I will merge this PR without actually launching the matlab plugin by myself.
> 
> > I will test your PR and try to port the improvements from Octave to Matlab later when I have more free time.
> 
> If you are interested in porting the improvements from the Octave plugin to the Matlab plugin. Just create PRs to this repo. I'm not a matlab user and failed to install a trial Matlab for reviewing PRs. I wonder if @pireddag would help verify (actually using the matlab plugin) the correctness of other PRs(if any). Or you can provide screenshots to demonstrate your changes.

I will send you a description of tests and screenshots under Linux.

da-liii commented:
This PR has been merged.  Matthias-A-Steiner's improvements will be available in the next release of GNU TeXmacs.

Matthias-A-Steiner commented:
Thank you, Dary!

> Am 13.03.2021 um 15:58 schrieb Darcy Shen ***@***.***>:
> 
> 
> This PR has been merged. Matthias-A-Steiner's improvements will be available in the next release of GNU TeXmacs.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/texmacs/matlab/pull/1#issuecomment-798478979>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AO5K64UMOBB6UOOBJHAZCXTTDN4QJANCNFSM4YT2DAGQ>.
>

pireddag commented:
> I will send you a description of tests and screenshots under Linux.

It works, I am just being a bit picky on the demo file :-) will send you some screenshots soon.
